lndmanage/lib/liquidityhints.py

```diff
@@ -17,12 +21,22 @@
 TIME_NODE_IS_SLOW_SEC = 5  # the time a node is viewed as slow
 
 
-class ShortChannelID(int):
-    pass
+@dataclass
+class AmountHistory:
+    amount: int = 0  # msat
```

Comment:
Would it make sense to initialize to `None`, in order to distinguish between undefined vs actual 0?

Reply:
Yes, that makes more sense! I changed it.